doc: add minutes for meeting 31 March 2022 (#1198)
nodejs/TSC: meetings/2022-03-31.md

# Node.js Technical Steering Committee (TSC) Meeting 2022-03-31

## Links

* **Recording**:  <http://www.youtube.com/watch?v=DFlvxeDa_nQ>
* **GitHub Issue**: <https://github.com/nodejs/TSC/issues/1196>

## Present

*Antoine du Hamel @aduh95 (TSC)
*Anatoli Papirovski @apapirovski (TSC)
*Beth Griggs @BethGriggs (TSC)
*Colin Ihrig @cjihrig (TSC)
*Gireesh Punathil @gireeshpunathil (TSC)
*Matteo Collina @mcollina (TSC)
*Michael Dawson @mhdawson (TSC)
*Darshan Sen @RaisinTen (TSC)
*Richard Lau @richardlau (TSC)
*Robert Nagy @ronag (TSC)
*Tobias Nießen @tniessen (TSC)
*Rich Trott @Trott (TSC)

## Agenda

### Announcements

> * Richard, release proposal for final 12.x release. Also reminder that it goes EOL

Reviewer: ```suggestion
* Richard, release proposal for final 12.x release, v12.22.12, <https://github.com/nodejs/node/pull/42531> with release candidate binaries available. Also reminder that it goes EOL
```

  at the end of April so make sure you are planning to move off.
* Rich, 18.x coming out in a few weeks as well.

### CPC and Board Meeting Updates

*Extracted from **tsc-agenda** labeled issues and pull requests from the **nodejs org** prior to the meeting.

* CPC update - no updates this week. Elections are coming up soon.
* Board meeting - Nothing of note this week.

### nodejs/node

* export v8 trace API [#42462](https://github.com/nodejs/node/pull/42462)

>   * James asked that we defer this to a meeting that he can make.

Reviewer: Sorry, I got confused with this one and the "Publishing specs for `Buffer` and `EventEmitter`" issue, which is the one James asked to defer, not this one.

    tagged was a bug fix to that.
  * Colin -> should move back to test_runner top level module, no conflict
    and consistent with current practice.  Avoids confusion, security issues
    of people installing the wrong thing.
  * Antoine in favor of enforcing the node: scheme for all new modules.
    Better decision long term
  * Biggest issue is with `node:` scheme is name confusion.
  * Currently needs to be SemVer major if we don’t use the `node:` scheme.
    But we could prevent loading if module available through `node:` scheme
    and throw an error.
  * Tobias, asked about conflict between the names, issue is though that people
    could be confused and load foo
  * Richard, if we are not going to allow userland modules with the same namespace
    then what is the point.
  * Antoine, long term plan would be to have all modules under the `node:` scheme
    but don’t need to force that.
  * Michael, since it can happen already due to namespaces, how much risk does it
    add?
  * Richard, could we add a flag which prevented it.
  * Antoine, if we use `node:` scheme is breaking change as it required changes
    in test scheme.

>   * Rich lets take back to the issue tracker, and commit to make a decision in the

Reviewer 2: ```suggestion
  * Rich: Let's take this back to the issue tracker, and commit to make a decision in the
```

>     Next 7 days.

Reviewer 2: ```suggestion
    next 7 days.
```
